adage/adage/settings.py

```diff
@@ -23,10 +24,17 @@
     CONFIG = CI_CONFIG
     pg_user = os.environ.get('PG_USER')
     pg_pass = os.environ.get('PG_PASSWORD')
-    CONFIG['dbmaster']['USER'] = pg_user
-    CONFIG['dbmaster']['PASSWORD'] = pg_pass
     CONFIG['databases']['default']['USER'] = pg_user
     CONFIG['databases']['default']['PASSWORD'] = pg_pass
+elif os.environ.get('CIRCLECI') == 'true':
+    with open(os.path.join(BASE_DIR, 'adage', 'config.py.template')) as f:
+        exec f
+    CONFIG = CI_CONFIG
+    CONFIG['databases']['default']['NAME'] = 'circleci_test'
+    CONFIG['databases']['default']['USER'] = 'ubuntu'
+    CONFIG['databases']['default']['PASSWORD'] = ''
```

Comment:
So... does user `ubuntu` automatically have access to the `circleci_test` database with no password? I don't see where this is set up.